### DISCUSSION TITLE
add swipe_up and Swipe_down events to the GestureDetector

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#2866] [Q&A]
I'm looking for the right event to do swipe_up and swipe_down but I can't find it in the docs. maybe that can be requested?
like tiktok scrolling for change page

**Comment by yunguangli** [comment]
on_vertical_drag_start and on_vertical_drag_update  can do the trick.  use various deltas in the event handler to decide if it is up or down.  check the GestureDetector control documentation.
